sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -1450,8 +1450,7 @@ case class ParseToDate(left: Expression, format: Option[Expression], child: Expr
   extends RuntimeReplaceable {
 
   def this(left: Expression, format: Expression) {
-      this(left, Option(format),
-        Cast(SecondsToTimestamp(UnixTimestamp(left, format)), DateType))
```

Comment:
Hi, @MaxGekk . If you don't mind, please make a JIRA issue for this. That will give us a traceability for this.

Reply:
Yeah, I was thinking about it too. Maybe it's best to file a JIRA

Reply:
done